rebar.config

```diff
@@ -44,13 +44,13 @@
 ]}.
 
 {pre_hooks, [
-	{"(linux)", compile, "env AR=gcc-ar make -C apps/arweave/lib/RandomX"},
-	{"(darwin)", compile, "make -C apps/arweave/lib/RandomX"},
+	{"(linux|darwin)", compile, "bash -c \"mkdir -p apps/arweave/lib/RandomX/build && cd apps/arweave/lib/RandomX/build && cmake .. > /dev/null\""},
+	{"(linux|darwin)", compile, "make -C apps/arweave/lib/RandomX/build"},
 	{"(linux)", compile, "env AR=gcc-ar make -C apps/arweave/c_src"},
```

Comment:
Does this ever get matched now?

Reply:
All steps are executed independently from whether any of the previous steps matched. In this case 2 steps are executed on both linux and darwin, and the third step is different for these architectures.